### PR TITLE
TokenizingByCharacters export to Onnx

### PR DESCRIPTION
**Pull request by Lynx1820**
- Transformer that tokenizes by character and returns the characters (as uint16)
- Since there's not a comparable onnx operator, a label encoder is used to map a string token to it's corresponding character value.  This will unfortunately make the model much larger, since 65535 values have to be saved as a mapping guide for label encoder.

### CODE REVIEW

**Comment by harishsk** [comment]
>     /// | Exportable to ONNX | No |

Please change this line whenever you add new support

---
Refers to: src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs:610 in 0532330. [](commit_id = 05323306729624a34cc53579eac0ee6a993017f7, deletion_comment = False)

File: test/Microsoft.ML.Tests/OnnxConversionTest.cs [hunk 1]
```diff
@@ -921,6 +921,36 @@ public void WordEmbeddingsTest()
...
+        public void TokenizingByCharactersOnnxConversionTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-test.tsv");
+            var dataView = ML.Data.LoadFromTextFile(dataPath, new[] {
+                new TextLoader.Column("label", DataKind.Boolean, 0),
+                new TextLoader.Column("text", DataKind.String, 1)
+            }, hasHeader: true);
+            var pipeline = new TokenizingByCharactersEstimator(mlContext, columns: new[] { ("TokenizedText", "text") });
+            var model = pipeline.Fit(dataView);
```
Comment (harishsk) [inline test/Microsoft.ML.Tests/OnnxConversionTest.cs]: Can you please add a test with UseMarkerChars?

File: test/Microsoft.ML.Tests/OnnxConversionTest.cs [hunk 2]
```diff
@@ -921,6 +921,36 @@ public void WordEmbeddingsTest()
             Done();
         }
 
+        [Fact]
+        public void TokenizingByCharactersOnnxConversionTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-test.tsv");
+            var dataView = ML.Data.LoadFromTextFile(dataPath, new[] {
```
Comment (harishsk) [inline test/Microsoft.ML.Tests/OnnxConversionTest.cs]: Are there no issues with utf-8 encoding here?

Comment (Lynx1820) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs]: yes, the issues happen with the wikipedia-detox-250-line-data.tsv file

Comment (harishsk) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs]: Does the test run successfully? If not is there a different file you can use? Or just used fixed inline data.

---
In reply to: [376146626](https://github.com/dotnet/machinelearning/pull/4805#discussion_r376146626) [](ancestors = 376146626)

Comment (Lynx1820) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs]: Yes, I'm using wikipedia-detox-250-line-**test**.tsv instead

Comment (harishsk) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs]: Ah..ok. Sounds good.

**Pull request merged**